package.json

```diff
@@ -31,8 +31,9 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@metamask/base-controller": "^3.0.0",
-    "@metamask/controller-utils": "^4.0.0",
+    "@metamask/base-controller": "^3.2.1",
+    "@metamask/controller-utils": "^5.0.0",
+    "@metamask/network-controller": "^15.2.0",
```

Comment:
[nit] Not strictly necessary but all of these have newer versions available now.